.gitignore

```diff
@@ -8,3 +8,4 @@ pom.xml.asc
 /.nrepl-port
 .hgignore
 .hg/
+.DS_Store
```

Comment:
Personally I'm of the opinion that only files specific to a given project should be included in its `.gitignore` and OS-specific/more general ignores should go in a global `.cvsignore`/`.gitignore`. But seeing that someone already included the Mercurial stuff before you, I suppose this is this project's convention.